static/scss/form.scss

```diff
@@ -8,6 +8,19 @@
       display: block;
     }
   }
+
+  select {
+    background: #f8f8f8;
+    border: 1px solid #bbb;
+    font-size: 15px;
+    color: rgba(0, 0, 0, 0.87);
+    border-radius: $std-border-radius;
+    height: 39px;
+    padding: 8px 20px;
+
+    /* If you add too much padding here, the options won't show in IE */
```

Comment:
I think this comment is supposed to go with the `padding` rule above, so could you move it to reflect that?